src/server/scripts/Northrend/ChamberOfAspects/RubySanctum/boss_halion.cpp

```diff
@@ -546,27 +494,53 @@ class boss_twilight_halion : public CreatureScript
             {
                 switch (spell->Id)
                 {
-                    case SPELL_TWILIGHT_DIVISION:
-                        if (Creature* controller = ObjectAccessor::GetCreature(*me, instance->GetGuidData(DATA_HALION_CONTROLLER)))
-                            controller->AI()->DoAction(ACTION_MONITOR_CORPOREALITY);
-                        break;
-                    default:
-                        generic_halionAI::SpellHit(who, spell);
```

Comment:
/home/travis/build/TrinityCore/TrinityCore/src/server/scripts/Northrend/ChamberOfAspects/RubySanctum/boss_halion.cpp:493:33: fatal error: 
```cpp
      unused parameter 'who' [-Wunused-parameter]

            void SpellHit(Unit* who, SpellInfo const* spell) override
                                ^
```
You need to look at line 493, which is 9 lines above this one, `generic_halionAI::SpellHit(who, spell);`